app/src/main/res/values/strings.xml

```diff
@@ -13,6 +13,9 @@
     <string name="general_label_save">Save</string>
     <string name="general_label_restart_workflow">Restart Workflow</string>
     <string name="general_label_ok">OK</string>
+    <string name="qr_code_enter_name_of_qr_code">Enter name of QR Code</string>
```

Comment:
you added four new strings, but looks like one is used. Please remove the rest

Reply:
Resolved.